### PR TITLE
Add SimTLMem

### PR DESCRIPTION
**Pull request by joey0320**
Add a SimTLMem which can be used in the simulation TestHarness when using a TL punch through

### CODE REVIEW

File: src/main/scala/SimTLMem.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+*/
+class SimTLMem(edge: TLEdgeParameters, size: BigInt, base: BigInt = 0)(implicit p: Parameters) extends SimpleLazyModule {
+  val node = TLClientNode(List(edge.master))
+  val srams = AddressSet.misaligned(base, size).map { aSet =>
+    LazyModule(new TLRAM(
+      address = aSet,
+      beatBytes = edge.bundle.dataBits/8))
+  }
+  val xbar = TLXbar()
+// srams.foreach{ s => s.node := TLBuffer() := TLFragmenter() := xbar }
```
Comment (jerryz123) [inline src/main/scala/SimTLMem.scala:19]: Remove the commented line

**Review by jerryz123 — CHANGES_REQUESTED** [review]
Can you rebase this against master chipyard and testchipip to capture the chisel3.5.6 bump?

File: src/main/scala/SimTLMem.scala (hunk 1 above)
Comment (joey0320) [reply src/main/scala/SimTLMem.scala]: Yes sir

File: src/main/scala/SimTLMem.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package testchipip
+
+import freechips.rocketchip.config.Parameters
```
Comment (jerryz123) [inline src/main/scala/SimTLMem.scala:3]: This needs to be org.chipsalliance.cde.config.Parameters

**Review by jerryz123 — CHANGES_REQUESTED** [review]

**Review by jerryz123 — APPROVED** [review]
Can you rebase this against master chipyard and testchipip to capture the chisel3.5.6 bump?

**Pull request merged**